src/js/claims-status/components/appeals-v2/CurrentStatus.jsx

```diff
@@ -2,17 +2,22 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const CurrentStatus = ({ title, description }) => (
-  <li className="process-step section-current">
+  <div className="current-status">
     <h2>Current Status</h2>
-    <h4>{title}</h4>
-    <div>{description}</div>
-  </li>
+    <div className="current-status-content">
+      <h3>{title}</h3>
+      <p>{description}</p>
```

Comment:
If `getStatusContents` returns an element for the description (because the appeal type is `bvaDecision`, for instance), we'll have
```html
<p>
  <div></div>
</p>
```
and React will yell at us again for the invalid DOM nesting.

Maybe do something more like:
```jsx
const descElement = typeof description === 'string' ? (<p>{description}</p>) : description;
```

Reply:
🤔 yeah agreed, and I don't want to introduce any more of these bugs than there already are. Seems like the safest thing to do might be to change this to a div, and on the `getStatusContents` side have each description output a jsx snippet instead of a string or snippet. That would let us enforce styles on the wrapping div, but have a semantic nested element with the actual content. Most of the content is already jsx snippets anyway.